### ISSUE TITLE
Remove previous plus/remove events

### ISSUE THREAD

**Issue by faritka** [unclead/yii2-multiple-input#55]
When I use AJAX to load a form with the multiple input element in a modal popup, the script attaches events to $(document) for the plus and minus buttons multiple times. When I click on the plus button after loading the form the second time, it adds two fields instead of just one.

Please remove the previous events.

```javascript
--- jquery.multipleInput.js     2015-11-11 13:06:09.067983422 -0800
+++ jquery.multipleInput.js.new 2015-11-11 13:03:33.743983347 -0800
@@ -77,11 +77,13 @@
                 attributeDefaults: {}
             });
 
+            $(document).off('click.multipleInput', '#' + settings.id + ' .js-input-remove');
             $(document).on('click.multipleInput', '#' + settings.id + ' .js-input-remove', function (e) {
                 e.preventDefault();
                 methods.removeInput.apply(this);
             });
 
+            $(document).off('click.multipleInput', '#' + settings.id + ' .js-input-plus');
             $(document).on('click.multipleInput', '#' + settings.id + ' .js-input-plus', function (e) {
                 e.preventDefault();
                 methods.addInput.apply(this);

```

**Comment by unclead** [comment]
It makes sense. Why dont you make a PR?

**Comment by faritka** [comment]
It'll probably be even better to attach events to a form instead of $(document).
Then when a form is removed, all events will be also removed. Now, they are left attached to the document object.

**Comment by unclead** [comment]
Closed via 9d879866f6498a6dc785bb8ac09f9961e76beb37
